tests/util/test_sendPREMISEvent.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from mock import Mock
+from urllib2 import URLError
+
+from codalib import util
+
+
+def test_is_successful(monkeypatch):
+    """
+    Check that sendPremisEvent returns the response and content.
+
+    Verify the conditions which will cause no exceptions to be
```

Comment:
I would suggest removing `which`, otherwise it seems as though you are trying to find the conditions that will cause no exceptions, rather than verifying that _these_ conditions cause no exceptions.